src/p5xr/core/p5xrButton.js

```diff
@@ -383,7 +383,9 @@ export default class p5xrButton {
  */
   __onXRButtonClick() {
     if (this.session) {
-      this.options.onEndSession(this.session);
+      // no longer end session on click
+      this.options.onRequestSession(this.device)
+      // this.options.onEndSession(this.session);
```

Comment:
This can be removed. See other comments.

Reply:
Removed b9994f7714ca755a121d0d34d2b5411eaf167e4e